site/cds_rdm/tasks.py

```diff
@@ -125,7 +125,11 @@ def sync_local_accounts_to_names(since=None, user_id=None):
                         f"Names sync | Unlisting CERN name for user {user.id}."
                     )
                     names_utils.update_name(user, cern_name, unlist=True, uow=uow)
-
+                else:
+                    current_app.logger.debug(
+                        f"Names sync | No CERN name found for user {user.id}. Creating new unlisted name."
```

Comment:
sorry I am a bit confused by the message. What is the use case for creating unlisted CERN name? is it when we have orcid entry but no related CERN user was found? why to create a CERN name at all?
how I understand the case when it is not needed to create unlisted:
1. I get an ORCID entry (from the dump). Most likely not a CERN researcher
2. I search for CERN user, I find none, because it is not a CERN person
3. I create a CERN name anyway, unlisted?

please let if I understood incorrectly

Reply:
Based on the discussion we had (we didn't specify this concrete case in the [issue](https://github.com/CERNDocumentServer/cds-rdm/issues/284) but the third point more or less describes it), I remember that we agreed to remove the person id from the `internal_id` but we keep the name when syncing it with ORCID value to have some sort of "history" and I guess it will be required for the migration, as we will match the unlisted value and not the ORCID value since there is no person id there anymore.

From now on, every time we sync if we find the ORCID value we create the unlisted CERN value as well, before it was simply merging to the ORCID value.

Reply:
ok I think I understand now.
My initial wrong assumption was that ORCID dump is the source of truth, but now I realise you iterate on user profiles. Makes sense now.
Can you add a comment above this log message, explaining why we do this unlisted name creation?